index.md

```diff
@@ -88,11 +88,13 @@ navigation:
         <div class="tab-content py-5 spark-install" id="nav-tabContent">
             <div class="tab-pane fade show active" id="nav-python" role="tabpanel" aria-labelledby="nav-python-tab">
                 <div class="mb-2 title">Run now</div>
-                <div style="font-size: 16px;">Installing with 'pip'
+                <div style="font-size: 16px;">Install with 'pip' or try offical image
                 </div>
                 <div class="code">
                     <p>$ pip install pyspark</p>
                     <p>$ pyspark</p>
+                    <p>$ </p>
+                    <p>$ docker run -it --rm spark:python3 /opt/spark/bin/pyspark</p>
```

Comment:
@zhengruifeng I feel this "try official image" is a bit unclear. Maybe we can reformat to this?
```
Install with pip
<code block>
Install with docker
<code block>
```

Reply:
make sense, will send a follow up pr